Week3/homework/ToDoApp_dump.sql

```diff
@@ -0,0 +1,113 @@
+/*!40000 ALTER TABLE `todos` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `users`
+--
+
+DROP TABLE IF EXISTS `users`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+ SET character_set_client = utf8mb4 ;
+CREATE TABLE `users` (
+  `User_Id` varchar(50) NOT NULL,
+  `Name` varchar(50) NOT NULL,
+  `Email` varchar(50) NOT NULL,
```

Comment:
**Non-blocking**

Email can be longer than 50:
https://7php.com/the-maximum-length-limit-of-an-email-address-is-254-not-320/

Reply:
Yes, I read the article in the link it has to be 254 chars I think.